packages/common/src/scopeSupportFacets/swift.ts

```diff
@@ -0,0 +1,188 @@
+import type { LanguageScopeSupportFacetMap } from "./scopeSupportFacets.types";
+import { ScopeSupportFacetLevel } from "./scopeSupportFacets.types";
+
+const { supported, unsupported, notApplicable } = ScopeSupportFacetLevel;
+
+export const swiftScopeSupport: LanguageScopeSupportFacetMap = {
+  // Command related
```

Comment:
I prefer to group the facets after supported/unsupported/notApplicable and in death order. 
See javascript for example
https://github.com/cursorless-dev/cursorless/blob/e5e9929c05e2607cc7bbddf1d22013b5aa89c772/packages/common/src/scopeSupportFacets/javascript.ts#L7